progress_stats/graphs.py

```diff
@@ -24,11 +26,27 @@
 
 _num_graphs = 0
 
-
-def progressGraphs(*args, **kwargs):
-  self = args[0]
-  old = kwargs['_old']
-
+def report_alt(self, type=0):
```

Comment:
This doesn't seem right.  This seems to be overriding the report method with copy-and-pasted code, which is brittle as Anki code could change in the future.